src/betterproto/plugin/parser.py

```diff
@@ -145,6 +152,24 @@ def generate_code(request: CodeGeneratorRequest) -> CodeGeneratorResponse:
     return response
 
 
+def _make_one_of_field_compiler(
+    output_package: OutputTemplate,
+    source_file: "FileDescriptorProto",
+    parent: MessageCompiler,
+    proto_obj: "FieldDescriptorProto",
+    path: List[int],
+) -> FieldCompiler:
```

Comment:
This shouldn't pass typechecking

Reply:
It does not; a lot of the code base, at the time of that fork, did not either:

```
src/betterproto/__init__.py:182: error: Argument "default" to "field" has incompatible type "Optional[Any]"; expected "Field[Any]"
src/betterproto/plugin/models.py:195: error: Incompatible types in assignment (expression has type "Union[Message, OutputTemplate]", variable has type "ProtoContentBase")
src/betterproto/plugin/models.py:202: error: Incompatible types in assignment (expression has type "Union[Message, OutputTemplate]", variable has type "ProtoContentBase")
src/betterproto/plugin/models.py:277: error: "str" has no attribute "name"
src/betterproto/plugin/models.py:294: error: Incompatible types in assignment (expression has type "Union[MessageCompiler, OutputTemplate]", base class "ProtoContentBase" defined the type as "Union[Message, OutputTemplate]")
src/betterproto/plugin/models.py:295: error: Incompatible types in assignment (expression has type "object", variable has type "DescriptorProto")
src/betterproto/plugin/models.py:296: error: Incompatible types in assignment (expression has type "object", variable has type "List[int]")
src/betterproto/plugin/models.py:323: error: "MessageCompiler" has no attribute "repeated"
src/betterproto/plugin/models.py:378: error: Incompatible types in assignment (expression has type "object", variable has type "MessageCompiler")
src/betterproto/plugin/models.py:379: error: Incompatible types in assignment (expression has type "FieldDescriptorProto", base class "MessageCompiler" defined the type as "DescriptorProto")
src/betterproto/plugin/models.py:461: error: Argument 2 to "is_map" has incompatible type "MessageCompiler"; expected "DescriptorProto"
src/betterproto/plugin/models.py:573: error: Incompatible types in assignment (expression has type "object", variable has type "Type[Any]")
src/betterproto/plugin/models.py:574: error: Incompatible types in assignment (expression has type "object", variable has type "Type[Any]")
src/betterproto/plugin/models.py:575: error: Incompatible types in assignment (expression has type "object", variable has type "str")
src/betterproto/plugin/models.py:576: error: Incompatible types in assignment (expression has type "object", variable has type "str")
src/betterproto/plugin/models.py:587: error: Incompatible types in assignment (expression has type "str", variable has type "Type[Any]")
src/betterproto/plugin/models.py:592: error: Incompatible types in assignment (expression has type "str", variable has type "Type[Any]")
src/betterproto/plugin/models.py:624: error: Incompatible types in assignment (expression has type "EnumDescriptorProto", base class "MessageCompiler" defined the type as "DescriptorProto")
src/betterproto/plugin/models.py:625: error: Incompatible types in assignment (expression has type "object", variable has type "List[EnumEntry]")
src/betterproto/plugin/models.py:660: error: Incompatible types in assignment (expression has type "object", variable has type "OutputTemplate")
src/betterproto/plugin/models.py:661: error: Incompatible types in assignment (expression has type "object", variable has type "DescriptorProto")
src/betterproto/plugin/models.py:662: error: Incompatible types in assignment (expression has type "object", variable has type "List[int]")
src/betterproto/plugin/models.py:683: error: Incompatible types in assignment (expression has type "ServiceCompiler", base class "ProtoContentBase" defined the type as "Union[Message, OutputTemplate]")
src/betterproto/plugin/models.py:685: error: Incompatible types in assignment (expression has type "object", variable has type "List[int]")
src/betterproto/plugin/parser.py:56: error: "Message" has no attribute "name"
src/betterproto/plugin/parser.py:57: error: Incompatible types in "yield" (actual type "Tuple[Message, List[int]]", expected type "Tuple[Union[EnumDescriptorProto, DescriptorProto], List[int]]")
src/betterproto/plugin/parser.py:84: error: Argument 1 to "append" of "list" has incompatible type "FileDescriptorProto"; expected "str"
src/betterproto/plugin/parser.py:99: error: Argument 1 to "traverse" has incompatible type "str"; expected "FileDescriptorProto"
src/betterproto/plugin/parser.py:101: error: Argument "source_file" to "read_protobuf_type" has incompatible type "str"; expected "FileDescriptorProto"
src/betterproto/plugin/parser.py:102: error: Argument "item" to "read_protobuf_type" has incompatible type "Union[EnumDescriptorProto, DescriptorProto]"; expected "DescriptorProto"
src/betterproto/plugin/parser.py:110: error: "str" has no attribute "service"
src/betterproto/plugin/parser.py:142: error: Incompatible types in assignment (expression has type "Path", variable has type "str")
src/betterproto/plugin/parser.py:195: error: Argument "proto_obj" to "ServiceCompiler" has incompatible type "ServiceDescriptorProto"; expected "DescriptorProto"
Found 33 errors in 3 files (checked 22 source files)
```